src/cclib/parser/orcaparser.py

```diff
@@ -62,6 +62,118 @@ def extract(self, inputfile, line):
         if "Program Version" in line:
             self.metadata["package_version"] = line.split()[2]
 
+        """
```

Comment:
Could you please use the format we typically have for these comments? Namely, start each line with`#` at the indentation level at which you are commenting. `"""` is meant for docstrings.